packages/openapi-v3/src/decorators/request-body.decorator.ts

```diff
@@ -109,7 +111,8 @@ export function requestBody(requestBodySpec?: Partial<RequestBodyObject>) {
       requestBodySpec[REQUEST_BODY_INDEX] = index;
     }
 
-    debug('  final spec: ', inspect(requestBodySpec, {depth: null}));
+    if (debug.enabled)
```

Comment:
Can you add a comment please to tell our code coverage tool to ignore the fact that this block is intentionally skipped by tests and thus should be not be counted towards code coverage?